### ISSUE TITLE
[Bug]: Does not accept color notations with spaces

### ISSUE THREAD

**Issue by kmoschcau** [tsparticles/tsparticles#5409]
### What happened?

This library does not accept color notations for the `colors` property, which have spaces in them. For example these do not work:
- `"rgb(255 255 255)"`
- `"oklch(70.3987% 0.141682 230.864678)"`

While these do work:
- `"rgb(255,255,255)"`
- `"#FFFFFF"`

### tsParticles Version

3.0.3

### tsParticles Configuration

```js
window.confetti({ colors: ["oklch(70.3987% 0.141682 230.864678)", "rgb(255,255,255)"] });
```

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matteobruni** [comment]
3.6.0 will accept the spaces as separator, but oklch is not supported yet. Colors are parsed by JavaScript to animate the particles, named colors doesn't work either (like black, white, cyan, aqua, or whatever).

**Comment by matteobruni** [comment]
Added plugin for OKLCH and LCH colors, but it's an external plugin available when 3.6.0 will be released
